src/nav/release-notes.yml

```diff
@@ -3,2958 +3,1968 @@ path: /docs/release-notes
 pages:
```

Comment:
This nav is automatically generated now, so it might be best for us to delete this and filter it out in our migration script.

Reply:
you mean remove all child pages from `nav/release-notes.yml` right? that way we still have a link to the release notes in the root nav i assume?

Reply:
updated migration script so it creates the `release-notes.yml` with no children pages so link still appears in root nav